src/site/content/en/learn/css/backgrounds/index.md

```diff
@@ -0,0 +1,456 @@
+---
+title: Backgrounds
+description: >
+  In this module learn the ways you can style backgrounds of CSS boxes using CSS
+audio:
+  title: The CSS Podcast - 053: Background
```

Comment:
This needs to go in quotes, since the extra ":" is confusing the YAML parser.

Reply:
```suggestion
  title: "The CSS Podcast - 053: Background"
```